app/screens/send-bitcoin-screen/confirm-destination-modal.tsx

```diff
@@ -124,31 +72,45 @@ export const ConfirmDestinationModal: React.FC<ConfirmDestinationModalProps> = (
               uncheckedIcon={"square-outline"}
               onPress={() => setConfirmationEnabled(!confirmationEnabled)}
             />
-            <Text style={styles.checkBoxText}>
+            <Text type={"p2"} style={styles.checkBoxText}>
               {LL.SendBitcoinDestinationScreen.confirmModal.checkBox({ lnAddress })}
             </Text>
           </View>
-        </View>
-        <Button
-          {...testProps(LL.SendBitcoinDestinationScreen.confirmModal.confirmButton())}
-          title={LL.SendBitcoinDestinationScreen.confirmModal.confirmButton()}
-          buttonStyle={styles.confirmButton}
-          onPress={confirmDestination}
-          disabled={!confirmationEnabled}
-          disabledStyle={styles.disabledConfirmButton}
-        />
-        <Button
-          title={LL.common.back()}
-          buttonStyle={styles.cancelButton}
-          titleStyle={styles.cancelButtonTitle}
-          onPress={() =>
-            dispatchDestinationStateAction({
-              type: "set-unparsed-destination",
-              payload: { unparsedDestination: destinationState.unparsedDestination },
-            })
-          }
-        />
+          <GaloyPrimaryButton
+            {...testProps(LL.SendBitcoinDestinationScreen.confirmModal.confirmButton())}
+            title={LL.SendBitcoinDestinationScreen.confirmModal.confirmButton()}
+            onPress={confirmDestination}
+            disabled={!confirmationEnabled}
+          />
+          <GaloySecondaryButton
```

Comment:
There needs to be space put between the GaloyPrimaryButton and GaloySecondaryButton, otherwise when you press the secondary button the button outline will show and it will be touching the primary button. You can set the `containerStyle` prop to accomplish this.

Reply:
ok will do in follow up PR